### DISCUSSION TITLE
The db seed does not work in my Nestjs sample prject

### DISCUSSION THREAD

**Discussion by hantsy** [prisma/prisma#7301] [Q&A] [answered]
I followed [this guide](https://www.prisma.io/docs/guides/database/seed-database) to add `seed.ts` to my Nestjs project, but it does not work when I ran `npx prisma db seed --preview-feature`.

```bash 
$ npx prisma db seed --preview-feature
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Running seed from prisma\seed.ts ...
```
No errors and no console log. 

The project source code is [here](https://github.com/hantsy/nestjs-prisma-sample).

**Comment by ryands17** [comment, marked as answer]
Remove the `published` field from the seed file and it will work fine. You should be getting an error as the `published` field is not present in the `Post` model.

**Reply by hantsy** [reply]
removed but the same result, there is no error reported in the console.

**Reply by ryands17** [reply]
I tried this same repo with Postgres 12. What version are you using?

**Reply by hantsy** [reply]
docker images
REPOSITORY   TAG       IMAGE ID       CREATED       SIZE
postgres     latest    b262c8b2fb54   12 days ago   315MB

**Reply by hantsy** [reply]
> starting PostgreSQL 13.3 (Debian 13.3-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit

**Reply by ryands17** [reply]
Could you check if the workaround in this issue fixes it?
https://github.com/prisma/prisma/issues/7176

**Reply by hantsy** [reply]
Change to array function, no luck.

**Reply by hantsy** [reply]
Maybe prisma can design an interface for developers, it is easy to force the developers to use the same form.
